btk/draw_blends.py

```diff
@@ -278,38 +293,45 @@ def run_mini_batch(self, blend_list, obs_conds):
             blend_list: List of catalogs with entries corresponding to one blend.
             obs_conds (list): List of `descwl.survey.Survey` class describing
                 observing conditions in different bands.
```

Comment:
Could you please update this docstring? I guess now a list of `btk.cutout.Cutout` objects